Resources/config/dbal.xml

```diff
@@ -64,6 +64,7 @@
             <argument>%doctrine.default_entity_manager%</argument>
         </service>
         <service id="Doctrine\Common\Persistence\ManagerRegistry" alias="doctrine" public="false" />
+        <service id="Symfony\Bridge\Doctrine\RegistryInterface" alias="doctrine" public="false" />
```

Comment:
The alias looks wrong here.

Reply:
Yea...  this is a bit unrelated. The issue is that the existing alias is a bit difficult, because there is an abstract class in the Bridge by the same name. It’s one of those situations where the user needs to guess the correct use statement, and only one will work. I’d like to add this ORM-specific alias and tell people to target it (but also mention the other interface, for those who care).